model.py

```
@@ -0,0 +1,141 @@
+    def init_weights(self):
+        for m in self.modules():
+            if isinstance(m, nn.Linear):
+                nn.init.kaiming_normal_(m.weight)
+                nn.init.zeros_(m.bias)
+
+class WideAndDeep(nn.Module):
+    def __init__(self,
+                 num_categories_dict: dict,  # 예: {'고객ID': 1000, '제품ID': 5000, ...}
+                 embedding_dim=8,
+                 num_numerical=8,  # 수치형 피처 개수
+                 hidden_dims=[128, 64],
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)